index.bs

```diff
@@ -474,6 +462,59 @@ To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [
         extensions, and I think it'll be more difficult to ship them after inverting the
         error-handling behavior.
 
+To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [=request=]
+(|request|) and a [=response=] (|response|), run these steps:
+
+1.  Let |embedder policy| be |request|'s [=request/client=]'s
+    [=environment settings object/embedder policy=].
+
+2.  If |request|'s [=request/reserved client=] is not `null`, then set |embedder policy|
+    to a new [=/embedder policy=].
+
+3.  If |embedder policy|'s [=embedder policy/report only reporting endpoint=] is not `null` and the
+    result of running [$cross-origin resource policy internal check] with
+    [=embedder policy/report only value=], |request| and |response| is `blocked`, then run these
+    steps:
+
+    1. Let |blocked url| be the first URL of |request|'s [=request/URL list=].
+
+    2. Set |blocked url|'s [=url/username=] to the empty string, and its [=url/password=] to `null`.
+
+    3. Set |serialized blocked url| be the result of executing the
+       [URL serializer](https://url.spec.whatwg.org/#concept-url-serializer) on |blocked url| with
+       the |exclude fragment flag| set.
+
+    4. Let |body| be a new object containing the following properties with keys:
```

Comment:
In CSP, we defined a [`CSPViolationReportBody`](https://w3c.github.io/webappsec-csp/#cspviolationreportbody) that (I hope!) created clear expectations for the bits and pieces of the reported data, both server-side and via `ReportingObserver` client-side. It might be reasonable to do the same here.

Reply:
Sounds reasonable. Can I do that separately?

Reply:
Yeah, doing this as a followup is fine (maybe add an ISSUE inline so we remember to do it?).

Reply:
Done. https://github.com/whatwg/html/issues/5328